packages/orderbook/src/order_provider/sra_polling_order_provider.ts

```diff
@@ -0,0 +1,79 @@
+import { assert } from '@0x/assert';
+
+import { OrderSet } from '../order_set';
+import { OrderStore } from '../order_store';
+import { SRAPollingOrderProviderOpts } from '../types';
+
+import { BaseSRAOrderProvider } from './base_sra_order_provider';
+
+const RECORD_COUNT = 100;
+
+export class SRAPollingOrderProvider extends BaseSRAOrderProvider {
+    private readonly _httpPollingIntervalId: Map<string, number> = new Map();
+    private readonly _pollingIntervalMs: number;
+
+    /**
+     * Instantiates a HTTP [Standard Relayer API](https://github.com/0xProject/standard-relayer-api)
+     * Polling Order Provider
+     * @param opts `SRAPollingOrderProviderOpts` containing the httpEndpoint to an SRA backend and polling options
+     * @param orderStore The `OrderStore` where orders are added and removed from
+     */
+    constructor(opts: SRAPollingOrderProviderOpts, orderStore: OrderStore) {
```

Comment:
When would the caller want to supply their own `orderStore`? Why not just instantiate one for them?

Reply:
One is instantiated in the higher level interface of `Orderbook`. This allows for a different persistent store like in the case of Launch Kit Frontend